### ISSUE TITLE
Install fails with "Helm check failed"

### ISSUE THREAD

**Issue by andresmgot** [vmware/dispatch#426]
### Expected behavior

I expected the install command to work on minikube.

### Current behavior

Installation command fails with:

```
Helm check failed: Error: Get http://localhost:8080/api/v1/namespaces/kube-system/configmaps?labelSelector=OWNER%!!(MISSING)D(MISSING)TILLER: dial tcp 127.0.0.1:8080: connect: connection refused
```

It also fails with the flag `--ignore-helm-check` with:

```
Error installing nginx-ingress chart: Error: UPGRADE FAILED: Get http://localhost:8080/api/v1/namespaces/kube-system/configmaps?labelSelector=NAME%!!(MISSING)D(MISSING)ingress%!!(MISSING)C(MISSING)OWNER%!!(MISSING)D(MISSING)TILLER%!!(MISSING)C(MISSING)STATUS%!!(MISSING)D(MISSING)DEPLOYED: dial tcp 127.0.0.1:8080: connect: connection refused
```

I have tried to delete the machine and do it again as explained in the troubleshoot guide but the result is the same.

[UPDATE]

I have tried with docker-for-mack (edge) as well. I keep receiving the same errors.

### Steps to reproduce

```
▶ minikube start --vm-driver=hyperkit --bootstrapper=kubeadm --kubernetes-version=v1.8.1

Starting local Kubernetes v1.8.1 cluster...
Starting VM...
Getting VM IP address...
Moving files into cluster...
Setting up certs...
Connecting to cluster...
Setting up kubeconfig...
Starting cluster components...
Kubectl is now configured to use the cluster.
Loading cached images from config file.

▶ export DISPATCH_HOST=$(minikube ip)

▶ cat << EOF > config.yaml
apiGateway:
  host: $DISPATCH_HOST
dispatch:
  host: $DISPATCH_HOST
  debug: true
  skipAuth: true
EOF

▶ helm init
$HELM_HOME has been configured at /Users/andresmartinez/.helm.

Tiller (the Helm server-side component) has been installed into your Kubernetes Cluster.

Please note: by default, Tiller is deployed with an insecure 'allow unauthenticated users' policy.
For more information on securing your installation see: https://docs.helm.sh/using_helm/#securing-your-helm-installation
Happy Helming!

▶ kubectl get pods -n kube-system -w
NAME                               READY     STATUS    RESTARTS   AGE
etcd-minikube                      1/1       Running   0          18m
kube-addon-manager-minikube        1/1       Running   0          16m
kube-apiserver-minikube            1/1       Running   0          16m
kube-controller-manager-minikube   1/1       Running   0          18m
kube-dns-545bc4bfd4-hkccf          3/3       Running   0          17m
kube-proxy-jjscj                   1/1       Running   0          17m
kube-scheduler-minikube            1/1       Running   0          18m
storage-provisioner                1/1       Running   0          17m
tiller-deploy-b6fbf658-vvjnp       1/1       Running   0          14s

▶ dispatch install --file config.yaml
Helm check failed: Error: Get http://localhost:8080/api/v1/namespaces/kube-system/configmaps?labelSelector=OWNER%!!(MISSING)D(MISSING)TILLER: dial tcp 127.0.0.1:8080: connect: connection refused
: exit status 1

▶ dispatch install --ignore-helm-check --file config.yaml
Creating new certificate for domain 192.168.64.6
Updating certificate in namespace dispatch
Creating new certificate for domain 192.168.64.6
Updating certificate in namespace kong
Installing nginx-ingress helm chart
Error installing nginx-ingress chart: Error: UPGRADE FAILED: Get http://localhost:8080/api/v1/namespaces/kube-system/configmaps?labelSelector=NAME%!!(MISSING)D(MISSING)ingress%!!(MISSING)C(MISSING)OWNER%!!(MISSING)D(MISSING)TILLER%!!(MISSING)C(MISSING)STATUS%!!(MISSING)D(MISSING)DEPLOYED: dial tcp 127.0.0.1:8080: connect: connection refused
: exit status 1
```

These are the versions I am using:
```
▶ minikube version
minikube version: v0.25.2
▶ helm version
Client: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}
▶ dispatch version
v0.1.13-d41f03a
```

### Impact

- [x] Low - Annoyance, but does not impact business or functionality
- [ ] Medium - Issue can be worked around, but is causing pain
- [ ] High - Blocker

### Your Environment

* Dispatch CLI version (or git commit): v0.1.13-d41f03a
* Dispatch Chart version (or image[s] tag):
* Operating System and version: OSX
* Kubernetes version (and distribution): minikube version: v0.25.2
* Etc (any other useful environmental information): Helm version v2.9.0

**Comment by kars7e** [comment]
Hi @andresmgot looks like you are hitting https://github.com/kubernetes/helm/issues/3985 which is a bug in helm 2.9.0. There is a fix scheduled for helm 2.9.1, in the meantime can you try with helm 2.8.2?

**Comment by andresmgot** [comment]
That worked, thanks!
